pkg/gofr/datasource/pubsub/nats/connectors.go

```diff
@@ -0,0 +1,24 @@
+// Package nats connector.go
+package nats
+
+import (
+	"github.com/nats-io/nats.go"
+	"github.com/nats-io/nats.go/jetstream"
+)
+
+type DefaultNATSConnector struct{}
```

Comment:
```suggestion
type DefaultConnector struct{}
```

You are already in nats package

Also it could be unexported

Reply:
sure done